src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -53,8 +54,7 @@
     "Microsoft.EntityFrameworkCore.Tools": {
       "version": "__ToolsPackageVersion__",
       "imports": [
-        "portable-net45+win8+dnxcore50",
-        "portable-net45+win8"
+        "portable-net451+win8"
```

Comment:
Are you sure this works? I've seen this pull in the wrong version of newtonsoft.net by not specifying +dnxcore50